docs/terminology.mdx

```diff
@@ -96,7 +96,8 @@ The full details of the request object aren't that important or helpful to know,
     {
       "type": "SsnCredential",
       "issuers": ["d14ca24c-5323-4b79-9dfa-6471b014d68c"],
-      "required": true
+      "required": true,
+      "description": "Most services that ask for SSN explain why they're asking for it, since people are nervous about providing it."
```

Comment:
I think this should be an example description to better match the rest of the example json.

```suggestion
      "description": "We use your social security number for identity verification and tax reporting."
```